BuildOptimizedImages/src/Web/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM microsoft/aspnetcore-build:1.0.1
+
+RUN mkdir /app
+WORKDIR /app
+
+ADD project.json .
+RUN dotnet restore --no-cache
```

Comment:
Is there a compelling reason to do --no-cache here? You are slowing down the restore for no real gain.